### PR TITLE
CASMINST-6574 CASMTRIAGE-6772 FAS new updates

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
# Description

Updated release notes with check_bios_firmware script issue.

Added issues in the known issues folder.

Added steps to Gigabyte updating M001 BMC/BIOS to check for completion.

Change to executable backup_smd_posgres.sh

PRs:
- CASMINST-6574
- CASMTRIAGE-6772

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by mbuchmann-hpe** [comment]
The two checks that are failing are on a script that all I changed was the permissions on to make it executable, will not be making any changes to the script as that would require revalidation of that script.

File: operations/firmware/Updating_Firmware_m001.md [hunk 1]
```diff
@@ -112,6 +122,30 @@ Use one of the following commands to find the model name for the node type in us
...
+    Using the return json packet, find the line:
+
+    ```text
+    "A new task /redfish/v1/TaskService/Tasks/596 was created."  
+    ```
+
+    Using the task number (596 in the above example), check the state of the task:
+
+    ```bash
+    curl -sku root:initial0 https://ipaddressOfBMC/redfish/v1/TaskService/Tasks/596 | jq .TaskState
```
Comment (jacobsalmela) [inline operations/firmware/Updating_Firmware_m001.md]: use generic creds

File: operations/firmware/Updating_Firmware_m001.md [hunk 2]
```diff
@@ -29,9 +29,6 @@ The following information is needed:
 
 Use one of the following commands to find the model name for the node type in use.
 
-- [Find HPE model name](#find-hpe-model-name)
-- [Find Gigabyte model name](#find-gigabyte-model-name)
```
Comment (jacobsalmela) [inline operations/firmware/Updating_Firmware_m001.md:33]: Is GB supposed to be removed here?

**Review by jacobsalmela — CHANGES_REQUESTED** [review]

File: operations/firmware/Updating_Firmware_m001.md (hunk 2 above)
Comment (mbuchmann-hpe) [reply operations/firmware/Updating_Firmware_m001.md:33]: Did not think links were needed for that small section so removed.

File: operations/firmware/Updating_Firmware_m001.md (hunk 1 above)
Comment (mbuchmann-hpe) [reply operations/firmware/Updating_Firmware_m001.md]: Updated - thanks

**Review by shunr-hpe — APPROVED** [review]

**Review by jwlv — APPROVED** [review]

**Pull request closed without merging**